Formula/gtk+.rb

```diff
@@ -61,6 +62,8 @@ def install
     end
     system "./configure", *args
     system "make", "install"
+
+    inreplace bin/"gtk-builder-convert", "#!/usr/bin/env python", "#!/usr/bin/python"
```

Comment:
should use regex since we don't want to replace '#!/usr/bin/env python3' if it becomes that

Reply:
Ah, fair point. Do you want them all changed?

Reply:
yes please. I'd use `%r{^#!/usr/bin/env python$}`

Reply:
Will do. I see the bot has flagged more issues too so I'll take a peek at those whilst I'm here.